src/main/java/net/openhft/chronicle/wire/TextWire.java

```diff
@@ -3136,14 +3136,12 @@ public Object typePrefixOrObject(Class tClass) {
                 bytes.readSkip(-1);
                 try {
                     return classLookup().forName(stringBuilder);
-                } catch (NoClassDefFoundError e) {
-                    throw new IORuntimeException("Unable to load class " + e, e);
```

Comment:
I also don't think we should be catching it.. 
> An Error is a subclass of Throwable that indicates serious problems that a reasonable application should not try to catch

Reply:
It appears to me,
The Error should be caught in ClassAliasPool.forName0 or not at all. c.f. the catch(ClassNotFoundException)
I assume this behaviour isn't consistent with other calls to forName().

Reply:
Yeah it's the only place it was done.

Reply:
So this was actually necessary when the case was wrong in the classname on Windows. I've moved the fix up to Core https://github.com/OpenHFT/Chronicle-Core/pull/373.